teps/0115-tekton-catalog-git-based-versioning.md

```diff
@@ -630,16 +593,64 @@ spec:
       value: 0.3
 ```
 
-#### HTTP Endpoint
+As we are working on migrating from the Tekton Hub to the Artifact Hub ([tektoncd/hub#667][667]), we propose adding an optional `"source"` parameter to the Hub Resolver indicating which Hub API should be used to fetch the catalog. The value can be set to `"artifact-hub"` or `"tekton-hub"`. The default value is `"artifact-hub"`.
```

Comment:
I think we will need to go a little bit more in detail in the migration path. 

- on the versionning, like `git-clone` on the new repository, what version does it starts on, etc (we discussed this a bit during a WG but we need to write it)
- on the one that will be host by a `tektoncd-*` and the others (and what happens to them)

Reply:
Good point! We can discuss further in this week's WG about the versions we want to migrate

Reply:
I have listed out the resource and versions to be migrated, PTAL 😁